src/views/users/User/UserCreateUpdate.vue

```diff
@@ -233,9 +235,17 @@ export default {
     disableMFAFieldIfNeed(user) {
       // SECURITY_MFA_AUTH 0 不开启 1 全局开启 2 管理员开启
       const adminUserIsNeed = (user?.is_superuser || user?.is_org_admin) && this.$route.meta.action === 'update' &&
-        store.getters.publicSettings['SECURITY_MFA_AUTH'] === 2
-      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === 1 || adminUserIsNeed) {
-        this.fieldsMeta['mfa_level'].disabled = true
+        store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.onlyAdminUsers
+      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.allUsers) {
+        this.fieldsMeta['mfa_level'].options = [{ 'value': MFALevel.allUsers, 'label': this.$t('MFAAllUsers') }]
+        this.initial.mfa_level = MFALevel.allUsers
+      }
+      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.onlyAdminUsers && adminUserIsNeed) {
+        this.fieldsMeta['mfa_level'].options = [{
+          'value': MFALevel.onlyAdminUsers,
+          'label': this.$t('MFAOnlyAdminUsers')
+        }]
+        this.initial.mfa_level = MFALevel.onlyAdminUsers
       }
     }
   }
```

Comment:
The code has no issues or potential issues at the provided knowledge cutoff point. It appears to be well written in accordance with Vue 3 and TypeScript standards. However, without reviewing it further since then, additional checks may be necessary.

As far as optimizations go:

1. `rules` property in `DataFormRules`.
2. Mapping array values instead of accessing them using `.map()`, potentially reducing memory access on objects of type `{...object}[]`.

Since we don't have more recent updates regarding security changes, there aren't clear areas where this would be an issue today either. This seems well-crafted considering these dates.

Please let me know if you need anything else checked!